readthedocs/proxito/middleware.py

```diff
@@ -159,7 +159,7 @@ def add_proxito_headers(self, request, response):
         if project_slug:
             cache_tags.append(project_slug)
         if version_slug:
-            cache_tags.append(f'{project_slug}-{version_slug}')
+            cache_tags.append(f'{project_slug}:{version_slug}')
```

Comment:
I think this change should be together with a comment here explaining why we decided to use `:` as a separator here.